policyengine_us/variables/gov/states/ct/tax/income/ct_income_tax_after_personal_credits.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_us.model_api import *
+
+
+class ct_income_tax_after_personal_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Connecticut income tax after personal tax credits and exemptions"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.CT
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("ct_agi", period)
+        personal_exemptions = tax_unit("ct_personal_exemptions", period)
+        income_after_personal_exemptions = max_(
```

Comment:
would this not be ct_taxable_income?

Reply:
Yes, I was not sure if they use the term multiple times through the tax form but it seems to fit here best (only time they use it)